tests/utility/v2/test_component_analyses.py

```diff
@@ -23,7 +23,7 @@
 from werkzeug.exceptions import BadRequest
 
 from bayesian.utility.v2.component_analyses import validate_version, \
-    known_package_flow, ca_validate_input, get_known_unknown_pkgs
+    known_package_flow, ca_validate_input, get_known_unknown_pkgs, normlize_packages
 
 
 class TestComponentAnalyses(unittest.TestCase):
```

Comment:
Can we add some test for two golang use-case below:
1. Package version with 'v' prefix like v1.2.3 as given version.
2. Package with incompatible version siffix like v2.3.4-alpha+incompatible.

This is to ensure that changes works for golang and create a safety net for future modifications.

Reply:
@dgpatelgit all requested Tests are covered in Utils in Previous PR..
I am leveraging same function.

Please find here written Tests https://github.com/fabric8-analytics/fabric8-analytics-utils/blob/master/tests/test_depencency_finder.py#L99

Reply:
lgtm